src/SelfAssessment/AssessmentQuestion.js

```diff
@@ -116,6 +117,16 @@ export const AssessmentQuestion = ({ onNext, onChange, option, question }) => {
     onChange(selectedValues)
   }, [selectedValues, onChange])
 
+  const handleOnNextPress = () => {
+    // if nothing is selected on multi question select none value from the survey json
```

Comment:
what do you think about removing this comment?

Reply:
Perhaps if it is very necessary, maybe we can extract the body of the clause into a function and we can name it something similar to the comment?

Reply:
I think since this is a small block of code there is no need to extract it into a different function. I will remove the comment, it's unnecessary.